xcube/core/store/store.py

```diff
@@ -123,93 +123,118 @@ def get_data_store_params_schema(cls) -> JsonObjectSchema:
 
     @classmethod
     @abstractmethod
-    def get_type_ids(cls) -> Tuple[str, ...]:
+    def get_type_specifiers(cls) -> Tuple[str, ...]:
         """
-        Get a tuple of supported data type identifiers.
+        Get a tuple of supported data type specifiers.
         The first entry in the tuple represents this store's default data type.
 
         :return: The tuple of supported data type identifiers.
         """
 
     @abstractmethod
-    def get_data_ids(self, type_id: str = None) -> Iterator[Tuple[str, Optional[str]]]:
+    def get_type_specifiers_for_data(self, data_id: str) -> Tuple[str, ...]:
```

Comment:
Please include a description of type specifiers and their usages into `DataStore`'s class documentation. Refer to that doc from the methods that have a `type_specifier` argument are return type specifiers.